src/logger.spec.js

```diff
@@ -3,20 +3,22 @@ const chalk = require('chalk');
 const { createMockInstance } = require('jest-create-mock-instance');
 const Logger = require('./logger');
 
+let logger;
 let outputStream;
-beforeEach(() => {
-    outputStream = createMockInstance(Writable);
-});
 
 const createLogger = options => {
     const logger = new Logger(Object.assign({ outputStream }, options));
     jest.spyOn(logger, 'log');
     return logger;
 };
 
-describe('#log()', () => {
+beforeEach(() => {
+    outputStream = createMockInstance(Writable);
+    logger = createLogger();
+});
+
+describe(Logger.prototype.log, () => {
```

Comment:
How do you think that this reads better?

I have been using `#` to indicate prototype instead of `.` for non-prototype properties for a while now. Don't remember where I got this from, maybe Java 😛 

The rest of the codebase is also using this standard, so it would deviate a bit, but if I'm convinced I can replace the other occurrences 👍

Reply:
In my experience, this description pattern helps validate that the method you are describing exists, and is updated appropriately when you make changes. An editor would validate that `Logger.prototype.log` exists, while it would _not_ validate a string being used in a description.